pyquest/drawer.py

```diff
@@ -279,163 +431,132 @@ def get_gate_rect_style(gate):
     return "solid"
 
 
-def get_gate_rect_color(gate):
-
-    # decoherence channels are red
-    if hasattr(pyquest.decoherence, type(gate).__name__):
-        return "red"
-
-    # initialisations are blue
-    if hasattr(pyquest.initialisations, type(gate).__name__):
-        return "blue"
-
-    # measurements are green
-    if isinstance(gate, M):
-        return "green"
-
-    # all other operators are black
-    return "black"
-
-
-def get_gate_face_color(gate):
-
-    # some specific operators have their own symbols and ergo colours
-    if isinstance(gate, Swap):
-        return "orange"
-    if isinstance(gate, Phase):
-        return "purple"
-
-    # generic gates have rectangle bodies with white faces
-    return "white"
-
-
-def get_vertical_connector_color(gate):
-
-    # some specific operators have their own connector colors
-    if isinstance(gate, Swap):
-        return "yellow"
-    if isinstance(gate, Phase):
-        return "pink"
-
-    # default
-    return "gray"
-
-
-def get_control_qubit_color(gate):
-
-    # phase gate controls should be indistinguishable from targets
-    if isinstance(gate, Phase):
-        return get_gate_face_color(gate)
-
-    return "black"
-
-
-def get_qubit_stave_color(qubit, num_qubits):
-
-    # qubit-specific stave colours... for some reason...
-    a, b = .05, .2
-    return colormaps['binary'](b + (a-b) * qubit/float(num_qubits))
-
-    # but I won't blame you for doing a boring fixed colour, like...
-    return "lightgray"
-
-
-
-'''
+"""
 Logic for producing graphics, which...
     - draws phase and control qubits as circles
     - makes decoherence channel borders dashed
     - draws swap gates with X symbols
     - labels gates with concise strings
     - merges gate bodies which target adjacent qubits
-'''
+    - draws target bullseye for CX and CXX
+"""
 
 
 def get_grouped_consecutive_items(nums):
 
     # [1,2,4,5,6] -> [(1,2), (4,5,6)]
     indAndNums = enumerate(sorted(nums))
-    for _, group in groupby(indAndNums, lambda x:x[0]-x[1]):
+    for _, group in groupby(indAndNums, lambda x: x[0] - x[1]):
         yield list(map(itemgetter(1), group))
 
 
 def get_gate_graphic_components(gate, column, num_qubits):
 
     # graphics consist of vertical connector lines, control circles, and gate body rectangles
-    lines = []        # item = [(x0,y0), (x1,y1)]
```

Comment:
(unrelated to the other consecutive comments)

Here you have reduced readability by forcing a convention that a robot might want of its code; to have constant spacing everywhere despite its harm to human readability. C'mon now!